sentry_sdk/integrations/wsgi.py

```diff
@@ -149,17 +149,22 @@ def get_client_ip(environ):
     return environ.get("REMOTE_ADDR")
 
 
-def _capture_exception(hub):
-    # type: (Hub) -> ExcInfo
+def _capture_exception(hub, e):
```

Comment:
Instead of taking an additional param here, could you read this info from `exc_info[2]`?